new_frontend/src/pages/archived/index.tsx

```
@@ -1,7 +1,172 @@
+          </form>
+        </div>
+      </div>
+
+      <div className="tw-flex tw-flex-col tw-w-full tw-flex-1 tw-overflow-y-auto tw-p-4 tw-gap-4">
+        {/* Initial loading skeletons */}
+        {isLoading && (
+          <>
+            {Array.from({ length: 5 }).map((_, i) => (
+              <NoticeCardSkeleton key={`loading-${i}`} />
+            ))}
+          </>
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)